apps/builder/app/builder/features/topbar/add-domain.tsx

```diff
@@ -95,7 +97,11 @@ export const AddDomain = ({
               autoFocus
               placeholder="your-domain.com"
               value={domain}
-              disabled={state !== "idle" || domainLoadingState !== "idle"}
+              disabled={
+                publishIsInProgress ||
+                state !== "idle" ||
```

Comment:
with all the states, it would make sense to call this one also something more specific, e.g. apiState or trpcState